### ISSUE TITLE
Dilato: Core: Linux only: It will take about 1 minute to appears the error message, when user change a filetree on network driver without network connection

### ISSUE THREAD

**Issue by ZuoBaogang** [adobe/brackets#14300]
Steps:
1. Launch Brackets
2. Open a folder on network driver
3. Disconnect the network
4. Try to change the filetree
5. Waiting about 1 minute

Result: After step 4, the Brackets change to dark, waiting about 1 minute, the error message appears.

Expect: The error message should appears immediately

![issue](https://user-images.githubusercontent.com/17721679/39626179-7a8d2d22-4fd3-11e8-948b-916d40c662f0.gif)

**Comment by ZuoBaogang** [comment]
System: Debian 9.1.0, Ununtu 17.10.1-64bit, Ununtu 16.04.3-32bit
Build: 1.13.17668

**Comment by boopeshmahendran** [comment]
@ZuoBaogang Does this issue happen with other file operations also like Rename and Delete? Can you check it?

**Comment by ZuoBaogang** [comment]
@boopeshmahendran 
Yes, this issue also happen with other file operations such as Reanme and Delete.
